### PR TITLE
Geniatech.TotalPlayer version 6.48.02

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98792)
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98792)

### CODE REVIEW

File: manifests/g/Geniatech/TotalPlayer/6.48.02/Geniatech.TotalPlayer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Geniatech.TotalPlayer
+PackageVersion: 6.48.02
+Installers:
+- InstallerUrl: https://www.geniatech.eu/wp-content/uploads/2017/04/Geniatech_TotalPlayer_6.48.02.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Geniatech_TotalPlayer_6.48.02.exe
+  InstallerSha256: 7598F216F100216A3F323A980B42BFB4F699E42BD7436E5D97EA98D210711E3E
```
Comment (stephengillie) [inline manifests/g/Geniatech/TotalPlayer/6.48.02/Geniatech.TotalPlayer.installer.yaml:13]:
```suggestion
  InstallerSha256: 7598F216F100216A3F323A980B42BFB4F699E42BD7436E5D97EA98D210711E3E
  InstallerSwitches:
    Silent: /s
```

**Comment by stephengillie** [comment]
Since the installer is InstallShield, the switch should be `/s` (or maybe `-s`). But this is complicated in testing by the package not installing correctly in the sandbox. It installs Visual C++ 2008 Redistributable, which has an issue:
image

This is suspiciously close to the message in #98696, so I'll be rechecking both in a full VM.

File: manifests/g/Geniatech/TotalPlayer/6.48.02/Geniatech.TotalPlayer.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/g/Geniatech/TotalPlayer/6.48.02/Geniatech.TotalPlayer.installer.yaml:13]: Testing the `/s` switch yields successful log entries but no shortcuts or other installed items. The switch `/S` has similar results, while other switches launch the interactive installer. I'm unsure which switch should be used here.

**Comment by stephengillie** [comment]
Unfortunately, the `/s` switch doesn't result in a completed install. The logs show a successful install, despite this. The attended installer required an email address, and I'm wondering if that's an issue here - maybe there's a secret installer switch to pass the email address - and maybe we can use something like `example@exmple.com` for this?

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**